### DISCUSSION TITLE
Row _id for newly submitted form

### DISCUSSION THREAD

**Discussion by turnerzzzz** [Budibase/budibase#14269] [Help] [answered]
Hi,

I have an automation that sends an email after a form has been saved. In the email template, there is a line that says "click here to open the ticket," and the word "here" contains a hyperlink to the ticket. However, for newly saved rows, I couldn't get the unique _id, though I can easily get it for existing rows.

Is there any way I can extract the unique row _id for newly saved rows?

Thank you!

**Comment by turnerzzzz** [comment]
Hello, any body knows?

**Comment by mikesealey** [comment, marked as answer]
Hi @timtandio 

If your automation is triggered by row created, you will have access to the bindings of the row that was created. 

For ease, I've got my automation set to send a discord message, but it will send anything that's available in the bindings drawer for that row. You will also be able to manually reference `trigger.row._id`, if you are specifically looking for `_id`
![image](https://github.com/user-attachments/assets/6a75e79b-e671-4167-beb8-80ae48773a55)
